crates/sui-core/src/consensus_adapter.rs

```diff
@@ -76,10 +81,9 @@ pub struct ConsensusAdapterMetrics {
     pub sequencing_fragment_control_delay: IntGauge,
 }
 
-const MAX_DELAY_MULTIPLIER: u64 = 100;
-fn weighted_average_half(old_average: u64, new_value: u64) -> u64 {
-    (500 * old_average + 500 * new_value) / 1000
-}
+/// Timeout for sequencing a certificate. The value is chosen to be high enough such that during normal operations,
+/// this timeout is not expected to be triggered.
+const SEQUENCING_CERTIFICATE_TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
Does SUI have a configuration file where this could be exported? It might be useful to be able to tune/change without a new commit here.

Reply:
Renamed the previous config option `delay_step` to `timeout_secs` for this purpose. There seems to be no override for `delay_step` in deployment configs.